### ISSUE TITLE
IES - backfacing hemisphere data is discarded

### ISSUE THREAD

**Issue by klayman** [LuxCoreRender/LuxCore#94]
Here is an IES file (renamed as .txt) which contains light distribution data for both hemispheres: [L2783.txt](https://github.com/LuxCoreRender/LuxCore/files/1897605/L2783.txt). The classic Lux uses the full data, while with LuxCore light is emitted only from the forward-facing side of mesh light.

It also would be nice to have an option to enable double-sided light emission. In case of using IES files, the "forward" hemisphere could be used for both directions, when double-sided emission is enabled.

**Comment by Dade916** [comment]
But it is the way old Lux was working too or have you found a difference ?

You need to use a point light source if you want light to be emitted backward (otherwise is blocked by the mesh surface).

**Comment by klayman** [comment]
The old Lux supports backward facing light distribution in IES files.

Lux classic:
![lux-classic](https://user-images.githubusercontent.com/350162/38688918-77890032-3ea4-11e8-9de5-403907570837.png)

LuxCore:
![luxcore](https://user-images.githubusercontent.com/350162/38688919-77b185de-3ea4-11e8-8dbe-425936224029.png)

Blend file:
[luxcorerender-double-sided-light.zip](https://github.com/LuxCoreRender/LuxCore/files/1903987/luxcorerender-double-sided-light.zip)

**Comment by Dade916** [comment]
I should have fixed the problem:

![ies](https://user-images.githubusercontent.com/3247881/38780404-e70c260e-40d5-11e8-8bd1-6460f482830e.jpg)

**Comment by klayman** [comment]
Great, thank you! Any plans for the double-sided light emission option (without IES)?

**Comment by Dade916** [comment]
Not at the moment but you can use an image instead of IES too. So if you use a 1x1 pixels white image, you will obtain the same result of a double sided are light (or constant IES map).
